pkgs/shells/zsh/zplug/default.nix

```diff
@@ -17,8 +17,8 @@ stdenv.mkDerivation rec {
   dontPatch = true;
 
   installPhase = ''
-    mkdir -p $out
-    cp -r $src/* $out/
+    mkdir -p $out/share/zplug
+    cp -r $src/* $out/share/zplug/
```

Comment:
Let's at least rescue `bin` to `/bin` and `doc/man` to `/share/man`...

Reply:
Good point. Done. `bin` is added to `/share/zplug/bin` instead of `/bin` because it will be added to `$PATH` via `$ZPLUG_ROOT/bin` in [zplug//base/core/core.zsh](https://github.com/zplug/zplug/blob/ac6c2a3e9eea6a488d96d98c752ef887e7d5aae3/base/core/core.zsh#L135).

I refer to the [AUR zplug package](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=zplug) for file list to copy.